tests/test_fit.py

```diff
@@ -25,14 +25,15 @@ class TestRobustFitting:
     def test_robust_norder_polynomial_fit(self, pkg_estimator: str) -> None:
 
         # Define x vector
-        x = np.linspace(1, 10, 1000)
+        x = np.linspace(-50, 50, 10000)
```

Comment:
This is to reduce random errors triggered on Mac sometimes (strange because `random_state` is fixed... Couldn't find a logical explanation).